modules/ROOT/pages/6.4.1-release-notes.adoc

```diff
@@ -0,0 +1,1138 @@
+//#TINY-9443
+
+In previous versions of {productname}, the searchreplace modal would close incorrectly when clicking outside of the alert dialog. The expected behavior in previous versions was that an error dialog would open when **no <string> match** was found within the {productname} editor viewport. The identifying issue found was when the user would `click` outside of the error dialog, resulting in the `Find and Replace` modal to closes incorrectly.
+
+In {productname} 6.4.1, when a **no <string> match** occurs within the {productname} editor viewport, an error banner will render in the `Find and Replace` modal beneath the (Replace with) field.
+
+As a result the when the instance occurs where **no <string> match** is triggered, the user will be prompted with the following warning: *`Could not find the specified string`*.
+
+=== Color picker on toolbar would not update when changing forecolor or backcolor from menu
+//#TINY-9439
+
+In {productname} 6.3, when the https://www.tiny.cloud/docs/tinymce/6/6.3-release-notes/#new-color_map_foreground-and-color_map_background-options-store-custom-colors-selected-using-the-text-color-and-background-color-menus[new color map] was introduced, a regression was found that affected the UI for *Color Swatch*.
+
+The issue occurred when a user would select a `forecolor` or `backcolor` via the `*Formats*` drop down menu, the color displayed on the color swatch toolbar buttons would not reflect the selected `forecolor` or `backcolor` respectively.
+
+In {productname} 6.4.1, the color in the relevant toolbar button is updated when changing either the `forecolor` or `backcolor` via a menu item.
+
+=== The onSetup api function would not run when defining custom group toolbar button
+//#TINY-9496
+
+A bug was discovered that affected the custom group toolbar button in {productname}, due to missing some event handlers. As a consequence, the `onSetup` api function would not run as intended when defining a custom group toolbar button.
+
+In {productname} 6.4.1, this bug was fixed by adding a handler to the toolbar group button that now calls the `api` as intended.
+
+As a result, the `onSetup` function for custom group toolbar buttons now runs correctly in {productname} 6.4.1.
+
+=== An element could be dropped onto the descendants of an element with a `contenteditable="false"` attribute
+//#TINY-9364
+
+In {productname} 6.0, the drag and drop `config` did not check some conditions surrounding element placements using either a drag or drop action.  This affected users running {productname} 6.0, which would effectively allow descendants of `contenteditable="false"` element to be dropped into another `contenteditable="false"` element.
+
+Fixes were made to ensure that `dragging | dropping` onto `contenteditable="false"` element targets on all browsers is `disabled`. This also includes when the drop target cursor is inside the `contenteditable="true"` element, or just before it.
```

Comment:
`dragging | dropping` doesn't seem appropriate here? just plain english "dragging or dropping"? something like that?

Reply:
```suggestion
Fixes were made to ensure that `dragging or dropping` onto `contenteditable="false"` element targets on all browsers is `disabled`. This also includes when the drop target cursor is inside the `contenteditable="true"` element, or just before it.
```